docs/src/nonparametric.md

```diff
@@ -69,6 +69,6 @@ ApproximateSignedRankTest
 ## Permutation test
 
 ```@docs
-ExactPermutationTest
-ApproximatePermutationTest
+PermutationTest(::Function, ::AbstractVector, ::AbstractVector)
+PermutationTest(::Function, ::AbstractVector, ::AbstractVector, ::Integer)
```

Comment:
Is this okay, or should `PermutationTest` have a single docstring that documents the exact and approximate variants?